rest/changes_api_test.go

```diff
@@ -198,6 +198,117 @@ func TestDocDeletionFromChannel(t *testing.T) {
 
 }
 
+//Simulate two SG instances to test rev_cache side effects
+func TestDocDeletionFromChannelTwoSG(t *testing.T) {
+	// See https://github.com/couchbase/couchbase-lite-ios/issues/59
+	// base.LogKeys["Changes"] = true
+	// base.LogKeys["Cache"] = true
+
+	rt1 := RestTester{SyncFn: `function(doc) {channel(doc.channel)}`}
+	defer rt1.Close()
+
+	a := rt1.ServerContext().Database("db").Authenticator()
+
+	// Create user in rt1:
+	alice, _ := a.NewUser("alice", "letmein", channels.SetOf("zero"))
+	a.Save(alice)
+
+	rt2 := RestTester{RestTesterBucket: rt1.RestTesterBucket, SyncFn: `function(doc) {channel(doc.channel)}`}
```

Comment:
I'm not familiar with `RestTester`, but will it follow the correct SG behaviour of storing the old rev with a TTL of 5 mins? If so, then the test isn't going to cover the long term behaviour... (if not is probably also an issue...)